.github/workflows/github-actions.yml

```diff
@@ -38,4 +38,11 @@ jobs:
           cd netzob/
           source venv/bin/activate
           sphinx-build -b html doc/documentation/source/ doc/documentation/build/
-      - run: echo "This job's status is ${{ job.status }}."
+      - name: Deploy pages
+        uses: peaceiris/actions-gh-pages@v3
```

Comment:
IMHO, you’d better use the official `deploy-pages` action from GitHub Actions: https://github.com/actions/deploy-pages